Backpack-SwiftUI/Font/Classes/Generated/BPKFontStyle.swift

```diff
@@ -28,126 +30,144 @@ public extension BPKFontStyle {
     
     /// Represents font size 16 with weight regular.
     static let bodyDefault = BPKFontStyle(
-        font: .regular(size:16),
+        font: .regular(size:16, textStyle:.body),
+        fontFixed: .regularFixed(size:16),
         letterSpacing: 0,
         lineHeight: 24
     )
 
     /// Represents font size 20 with weight regular.
     static let bodyLongform = BPKFontStyle(
-        font: .regular(size:20),
+        font: .regular(size:20, textStyle:.body),
+        fontFixed: .regularFixed(size:20),
         letterSpacing: 0,
         lineHeight: 28
     )
 
     /// Represents font size 12 with weight regular.
     static let caption = BPKFontStyle(
-        font: .regular(size:12),
+        font: .regular(size:12, textStyle:.caption),
```

Comment:
The linting here is wrong I think, theres no space between the `textStyle:.caption`? linter should have caught this tho :?